kivy/lang/parser.py

```diff
@@ -422,6 +422,10 @@ def execute_directives(self):
                     ref = ref[6:].strip()
                     force_load = True
 
+                # if #:include [force] "path with quotes around"
+                if ref[0] == ref[-1] and ref[0] in ('"', "'"):
+                    ref = ref[1:-1]
```

Comment:
But what was wrong with https://github.com/kivy/kivy/pull/5153/commits/ccb1f7da31e79856e3fb0da4706305857973cea4 ? That seems like the perfect solution. Triple quotes, although ugly is allowed on one line which that commit handles well, but not the latest commit.